src/TypeScript/Rules/ActiveRulesCalculator.cs

```diff
@@ -89,7 +89,9 @@ private static Rule Convert(RuleDefinition ruleDefinition)
                 Key = ruleDefinition.EslintKey,
 
                 // TODO: handle parameterised rules #2284
-                Configurations = ruleDefinition.DefaultParams
+                Configurations = ruleDefinition.DefaultParams,
+
+                FileTypeTarget = new []{"MAIN"}
```

Comment:
new mandatory field. Options are "MAIN" and "TEST", I'm not sure where we are supposed to take them from.